src/management-system-v2/app/(dashboard)/[environmentId]/processes/[processId]/modeler-toolbar.tsx

```diff
@@ -294,6 +296,11 @@ const ModelerToolbar = ({
                       Open Subprocess
                     </Button>
                   </Tooltip>
+                )) ||
+                (bpmnIs(selectedElement, 'bpmn:ScriptTask') && (
```

Comment:
The button is visible even when the environment flag enabling the execution features is not set.